operator/deploy/kiali/kiali_cr.yaml

```diff
@@ -552,7 +552,8 @@ spec:
 #
 # List of namespaces or regex defining namespaces to include in a cache.
 #    ---
-#    cache_namespaces: []
+#    cache_namespaces:
+#    - ".*"
```

Comment:
Is it missing the default value also in config.go perhaps ?

Reply:
good point, it's updated